lib/hashable.rb

```diff
@@ -0,0 +1,7 @@
+module Hashable
+
+  def key_with_max_value(hash)
+    hash.key(hash.values.max)
+  end
+
+end
```

Comment:
Decided to break this simple method out into a module because I was writing this code often and the names of the hashes were particularly long.

Reply:
Oh, good call. That's a great use of a module, imho.